keyboards/mmd/km75/readme.md

```diff
@@ -0,0 +1,22 @@
+# MMD-KM75
+
+
+* A customizable soldering 75% keyboard.
+
+* Keyboard Maintainer: [MMD](https://github.com/Smartmmd)
+* Hardware Supported: [MMD](https://github.com/Smartmmd)
+* Hardware Availability: [MMD](https://github.com/Smartmmd)
```

Comment:
There's no hardware at this location. Please update the links.

Reply:
updated  :-）